frontend/dev.Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:20
+
+WORKDIR /app
+
+EXPOSE 3000
+
+VOLUME [ "/app" ]
+
+ENTRYPOINT [ "npm", "run", "container-run:dev" ]
```

Comment:
Might be worth running `docker init` to generate this so that it does all the install and build steps